docs/deploy_logging.md

```diff
@@ -0,0 +1,49 @@
+# Deploy openshift-logging on cluster
+
+### Pre-requisites before starting to deploy openshift-logging
+
+###### It is recommended to have minimum m5.xlarge setup.
+
+The images for the openshift-logging need to be pulled from the quay,
+sync the images with the internal registry and update the cluster with the new images for containers.
+
+
+Note: Steps are to be followed till OCP 4.2 is GAed, These steps can be skipped after that
+
+1. Create a quay.io account [https://quay.io]
+
+2. Add your quay username in the sheet given below and wait for the invite sent
+from quay.io ocp-repo maintainers
+https://docs.google.com/spreadsheets/d/1OyUtbu9aiAi3rfkappz5gcq5FjUbMQtJG4jZCNqVT20/edit#gid=0
+
+3. Generate the quay_token with the command
+
+```
+curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
+{
+    "user": {
+        "username": "'"$USER"'",
+        "password": "'"$PASSWD"'"
+    }
+}' | jq -r '.token'
+```
+
+4.Save the token as quay.token under ocs-ci/data/
```

Comment:
if you are mentioning path of riles please surround them by \` . So it will look like `quay.token` which is better readable in text that this means some path of file or script or whatever.